cmd/cli/kubectl-kyverno/apply/apply_command.go

```diff
@@ -164,6 +171,7 @@ func Command() *cobra.Command {
 	cmd.Flags().BoolVarP(&applyCommandConfig.RegistryAccess, "registry", "", false, "If set to true, access the image registry using local docker credentials to populate external data")
 	cmd.Flags().StringVarP(&applyCommandConfig.KubeConfig, "kubeconfig", "", "", "path to kubeconfig file with authorization and master location information")
 	cmd.Flags().StringVarP(&applyCommandConfig.Context, "context", "", "", "The name of the kubeconfig context to use")
+	cmd.Flags().StringVarP(&applyCommandConfig.GitBranch, "git-branch", "b", "", "test github repository branch")
```

Comment:
Same thing here in this message. git branches aren't only from GitHub.

Reply:
So, what good name name can we give in place of `git-branch`, `GitBranch`. Any suggestion ?

Reply:
`test git repository branch` ?

Reply:
The flag is fine. The message needs to be fixed.

Reply:
Okay, go it.